x-pack/legacy/plugins/infra/public/components/logging/log_text_stream/log_entry_field_column.tsx

```diff
@@ -35,9 +31,12 @@ export const LogEntryFieldColumn: React.FunctionComponent<LogEntryFieldColumnPro
   isHovered,
   isWrapped,
 }) => {
-  const value = useMemo(() => (isFieldColumn(columnValue) ? JSON.parse(columnValue.value) : null), [
```

Comment:
We no longer `stringify` the values in the API level, so we don't need to `JSON.parse()` them here